test/minimum_eigensolvers/evqe/test_evqe_operators.py

```diff
@@ -0,0 +1,208 @@
+# Quantum Evolving Ansatz Variational Solver (QUEASARS)
+# Copyright 2023 DLR - Deutsches Zentrum für Luft- und Raumfahrt e.V.
+
+from typing import Optional
+
+import pytest
+from dask.distributed import LocalCluster, Client
+from docplex.mp.model import Model
+from qiskit_aer.primitives import Estimator
+from qiskit_algorithms.optimizers import NFT, Optimizer
+from qiskit_optimization.translators import from_docplex_mp, to_ising
+from qiskit_optimization.converters import IntegerToBinary
+from qiskit.quantum_info import Operator
+
+from queasars.circuit_evaluation.circuit_evaluation import OperatorCircuitEvaluator
+from queasars.minimum_eigensolvers.base.evolutionary_algorithm import OperatorContext, BasePopulationEvaluationResult
+from queasars.minimum_eigensolvers.evqe.evolutionary_algorithm.individual import EVQEIndividual
+from queasars.minimum_eigensolvers.evqe.evolutionary_algorithm.population import EVQEPopulation
+from queasars.minimum_eigensolvers.evqe.evolutionary_algorithm.mutation import (
+    EVQELastLayerParameterSearch,
+    EVQEParameterSearch,
+    EVQETopologicalSearch,
+    EVQELayerRemoval,
+)
+from queasars.minimum_eigensolvers.evqe.evolutionary_algorithm.speciation import EVQESpeciation
+from queasars.minimum_eigensolvers.evqe.evolutionary_algorithm.selection import EVQESelection
+
+
+class TestEVQEOperators:
+    client: Optional[Client] = None
+    evaluator: Optional[OperatorCircuitEvaluator] = None
+    operator: Optional[Operator] = None
```

Comment:
I think pytest can cache fixtures itself, you don't need to do this manually (if this was your intent and I didn't misunderstand anything): See https://docs.pytest.org/en/stable/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Is it actually necessary to reuse the fixtures? For the client I understand this because it's using an external resource (threads).

Reply:
I was not aware of this feature. Thank you! I applied this to appropriately create and destroy the dask cluster. The other fixtures are not cached anymore, as i think you are right, that it is not necessary for them.